sites/fast-creator/app/creator.props.ts

```diff
@@ -13,6 +13,10 @@ export enum NavigationId {
     libraries,
 }
 
+export enum DisplayMode {
+    default,
+    preview,
```

Comment:
We may have more display modes in the future, so we might want to stay away from "default". Also, assigning these to strings means that this is a little more future-proof, in case this is ever stored somewhere and these values get moved around for some reason.

```suggestion
    interactive = "interactive",
    preview = "preview",
```

Reply:
Done